tests/test_other.py

```diff
@@ -609,6 +610,14 @@ def test_emsize(self):
       output = self.run_process(cmd, stdout=PIPE).stdout
       self.assertContained(expected, output)
 
+  def test_emstrip(self):
+    self.run_process([EMCC, test_file('hello_world.c'), '-g', '-o', 'hello.js'])
+    output = self.run_process([LLVM_OBJDUMP, '-h', 'hello.wasm'], stdout=PIPE).stdout
+    self.assertContained('.debug_info', output)
+    self.run_process([emstrip, 'hello.wasm'])
+    output = self.run_process([LLVM_OBJDUMP, '-h', 'hello.wasm'], stdout=PIPE).stdout
+    self.assertContained('.debug_info', output)
```

Comment:
Should this be `assertNotContained`?

Reply:
lol yes and I wondered why this test was failing :rofl: